hack/build-rpm-release.sh

```diff
@@ -23,32 +23,33 @@ tito tag --use-version="${OS_RPM_VERSION}" \
          --no-auto-changelog --offline
 tito_tmp_dir="${BASETMPDIR}/tito"
 mkdir -p "${tito_tmp_dir}"
+tito build --offline --srpm --rpmbuild-options="--define 'dist .el7'" --output="${tito_tmp_dir}"
```

Comment:
I'm not familiar with SRPM too much -- but why does the distribution matter for what should be a compressed bundle of source files?